code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -449,36 +622,86 @@ There are several things that need to be remembered:
 
 //human HUD updates for items in our inventory
 
-//update whether our head item appears on our hud.
+/mob/living/carbon/human/proc/update_hud_uniform(obj/item/I)
```

Comment:
all of these hud procs should have better var names and auto docs (it's not 100% clear by the name)